packages/test/test-end-to-end-tests/src/test/gc/gcStateResetInSummaries.spec.ts

```diff
@@ -35,7 +35,7 @@ describeNoCompat("GC state reset in summaries", (getTestObjectProvider) => {
 	let provider: ITestObjectProvider;
 	let mainContainer: IContainer;
 	const settings = {
-		"Fluid.ContainerRuntime.Test.SummaryStateUpdateMethodV2": "refreshFromSnapshot",
+		"Fluid.ContainerRuntime.Test.CloseSummarizerDelayOverrideMs": 10,
```

Comment:
Why setting the delay override?

Reply:
Don't really want to wait 5 seconds for the test to end, it'll likely timeout.

Reply:
lol I saw this and thought it was 10 seconds.  Looks good.